src/test/java/com/amihaiemil/zold/RtWalletTestCase.java

```diff
@@ -0,0 +1,78 @@
+package com.amihaiemil.zold;
+
+import com.amihaiemil.zold.mock.AssertRequest;
+import com.amihaiemil.zold.mock.Condition;
+import com.amihaiemil.zold.mock.MockHttpClient;
+import com.amihaiemil.zold.mock.Response;
+import org.apache.hc.client5.http.classic.HttpClient;
+import org.apache.hc.core5.http.ClassicHttpResponse;
+import org.apache.hc.core5.http.ContentType;
+import org.apache.hc.core5.http.HttpStatus;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Provides a test case for the {@link RtWallet} operations.
+ */
+public class RtWalletTestCase {
+
+    /**
+     * A base URI for testing.
+     */
+    private static final String BASE_URI = "http://wts.test.com";
+
+    /**
+     * Checks whether the wallet can be instantiated.
+     */
+    @Test
+    public void instantiates() {
+        MatcherAssert.assertThat(
```

Comment:
@g4s8 , partially agree here. As far as we could not forecast possible future adjustments the way the `RtWallet` could be used in, I think it would make sense in performing such checks (by the way, there was already one of them before - com.amihaiemil.zold.RestfulWtsTestCase#isInstantiated).

Reply:
@g4s8 , I agree with you in removing the assertion. This has been corrected within the recent update.